src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/write/fluentd/FluentdWriter.java

```diff
@@ -233,7 +235,11 @@ private void emitData(String tag, Map<String, Object> data) throws IOException {
         while (true) {
             LOGGER.log(Level.FINEST, "Emitting data: Try {0} Data: {1}", new Object[] {count, data });
             try {
-                fluentd.emit(tag, data);
+                Instant instant = Instant.parse((String) data.get("timestamp"));
```

Comment:
Define a constant for the field name. I propose to the extract field name constants into a separate class.
Also move the DateTimeFormatter for the `timestamp` field there and use it here for parsing.

Reply:
Done in [e194596](https://github.com/SAP/elasticsearch-logs-plugin/pull/96/commits/e1945966caf328c68cf08caf60cc185779efdc7d).